Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -2228,6 +2226,10 @@ void ParseV3000BondBlock(std::istream *inStream, unsigned int &line,
         int reactStatus = FileParserUtils::toInt(val);
         bond->setProp("molReactStatus", reactStatus);
       } else if (prop == "STBOX") {
+      } else if (prop == "ENDPTS") {
+        bond->setProp(common_properties::_MolFileBondEndPts, val);
```

Comment:
Should there be a test case that verifies that what is in _MolFileBondEndPts is actually correct?

Reply:
Yep. Will add it